roles/add-networks/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: get node and master addresses
+  shell: "openstack coe cluster show {{ cluster_name }} -c master_addresses -c node_addresses -f json"
+  register: result_addresses
+
+- name: get list of servers
```

Comment:
Is it not possible to get the servers part of the COE from the COE CLI, rather than getting all servers here? It seems a little risky getting all servers, because what if you accidentally reconfigure one that isn't part of a COE?

Reply:
COE cli doesn't give us the host names. We need the host names in order to attach networks and we filter out the correct hostnames before making changes.

Reply:
Perhaps you could scope the server list with a user id, eg.```--user bharat```?

Reply:
Thats a good idea!